data/migrations/20211215202038_Submissions_Table.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable('Submissions-New', (t) => {
+      t.timestamp('startAt');
+      t.timestamp('readAt');
+      t.timestamp('drawAt');
+      t.timestamp('writeAt');
+      t.timestamp('squadUpAt');
+      t.timestamp('voteAt');
+
+      t.string('gameMode').notNullable();
+    })
+    .dropTable('Submissions-New')
+    .renameTable('Submissions-New', 'Submissions');
+};
+
+exports.down = function (knex) {
+  knex.schema.dropTableIfItExists('Submissions');
```

Comment:
I think the down migration has to reverse all the steps in up migration. 

So you need to do this

```js
renameTable("Submissions","Submissions-New")
```